RELEASING/README.md

```diff
@@ -74,6 +90,7 @@ Then you can generate other derived environment variables that are used
 throughout the release process:
 
 ```bash
+    export FULLNAME="Maxime Beauchemin"
```

Comment:
Probably best to avoid using values that can cause misleading results if copy-pasted as such. Also always good to avoid introducing unnecessary PII where not necessary, eve if only as an example. In this case I would probably write `YOURFULLNAMEHERE` or similar to make it obvious that it needs to be replaced.

Reply:
Agree, changed it and left a comment to give a little more context to it